.travis.yml

```
@@ -8,6 +8,9 @@ env:
   SECRET_KEY: sameple_secret_key
   ALLOWED_HOSTS: "*"
   DEBUG: 'True'
+  EMAIL_BACKEND: 'your-backend-config'
+  AWS_ACCESS_KEY_ID: 'AWS_ACCESS_KEY_ID'
+  AWS_SECRET_ACCESS_KEY: 'AWS_SECRET_ACCESS_KEY'
   EMAIL_HOST_USER: naveennvrgup@gmail.com
   EMAIL_HOST_PASSWORD: wnejszjzrvacvkpn
```

```suggestion
  MOCK_EMAIL: 'True'
```
Since mock email now available we don't need these envvar in the travis setup from now on

---

@naveennvrgup When I remove that envvar, the travis build fails, what should I do?